components/tool/ranking/component.js

```diff
@@ -1,8 +1,16 @@
 import React, { useMemo } from 'react';
 import PropTypes from 'prop-types';
+import { format } from 'd3-format';
 
 import './style.scss';
 
+// The units are sent only abbreviated, so we need to map them to their full names
+const units = {
+  ha: 'hectares',
+  t: 'tonnes',
```

Comment:
I'm not sure about this but it may be preferable to use “metric tons”. It may be clearer for Americans.